### DISCUSSION TITLE
Welcome to WalkingTourClient Discussions!

### DISCUSSION THREAD

**Discussion by JGL** [JGL/WalkingTourClient#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jkosem** [comment]
I'm Jim. Working with @JGL and @apluskowski to make the bestest way to connect wine, AR and castles.
